projects/kit/src/lib/masks/number/number-mask.ts

```diff
@@ -23,7 +23,7 @@ export function maskitoNumberOptionsGenerator({
     isNegativeAllowed = true,
     precision = 0,
     thousandSeparator = CHAR_NO_BREAK_SPACE,
-    decimalSeparator = ',',
+    decimalSeparator = '.',
```

Comment:
**Cypress:**
```
Number | Max validation
       Max = 3
         accepts any decimal value (integer part is less than max one)
           0,9999:

      Timed out retrying after 10100ms
      + expected - actual

      -'3'
      +'0.9999'
```

```
Number | Max validation
       Max = 3
         accepts any decimal value (integer part is less than max one)
           2,777:

      Timed out retrying after 10100ms
      + expected - actual

      -'3'
      +'2.777'
```

```
Number | Basic
       minus sign
         can type "—0,12" (em-dash):

      Timed out retrying after 10100ms
      + expected - actual

      -'−012'
      +'−0.12'
```

Reply:
Oh, the last one shows a bug – we can type many leading zeroes for negative number.

I will create issue for this case

Reply:
* https://github.com/Tinkoff/maskito/issues/194